routes/api/v1/playlists.js

```diff
@@ -29,6 +29,24 @@ router.post('/', async (request, response) => {
   }
 });
 
+router.get('/', async (request, response) => {
+  let playlists = await database('playlists').select();
+
+  if (playlists) {
+    let playlistsArray = playlists.map(playlist => {
+      return {
```

Comment:
Great use of an object literal here. I think that makes it a bit easier than instantiating a playlist object - even if it seems so backwards compared to Ruby.